_layouts/get-involved.html.slim

```diff
@@ -81,11 +81,11 @@ ul.small-block-grid-1.large-block-grid-4
     a(href ="#{site.base_url}/video/vimeo/92868336/" )
       img(src=cdn(site.base_url + "/images/people/MarkusEisele.jpg") alt="Markus Eisele")
     div.get-involved-label
-      p.title Java Champion
+      p.title Java Hero
       p.name Markus Eisele
     blockquote It's more about a working living healthy community.
 
-    p We spoke with Markus Eisele about his role as a Java Champion and the JBoss community.
+    p We spoke with Markus Eisele about his role as a Java Hero and the JBoss community.
```

Comment:
This needs to stay as 'Java Champion'. It's just the 'JBoss Champion' name that's changing.